### ISSUE TITLE
Stopped working after latest patch

### ISSUE THREAD

**Issue by De-Seppe** [SuiMachine/Doom-64-Sui-s-Hack#3]
The hack has stopped working after the latest patch.

**Comment by SuiMachine** [comment]
Yeah, that won't be working after the patch, cause the addresses of instructions changed. Is there even an interest in having Y axis mouse look still (considering they have fixed the damage values for Hell Knight?

**Comment by SoftDustyRhodes** [comment]
For me, yes. Without Y look I'd rather switch to a source port.

**Comment by SuiMachine** [comment]
Very well - will try and get it restored (again, it's just a look, not actually aiming). Thankfully they fixed other major issues in this patch.

**Comment by SuiMachine** [comment]
Welp, I've just spent like 2 hours trying to find it camera rotation and I just can't. Need smarter people (or people with more time on their hands) to look into it :(

**Comment by SoftDustyRhodes** [comment]
Well drat. Guess that's the end of that

**Comment by SuiMachine** [comment]
I guess there is not much point in working on it now:
https://steamcommunity.com/app/1148590/discussions/0/3193619419610701034/

Yeah, I also noticed FPS issues on my PC. So will probably wait until they fix it.

**Comment by Zorkats** [comment]
Apparently they are working on a new patch to fix the issues. And the 91 fps cap is apparently a leftover from the stadia version... an oopsie doopsie from the devs i guess haha...
